<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>department</class>
 <widget class="QWidget" name="department">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Departments Managment</string>
  </property>
  <widget class="QGroupBox" name="Big">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Insert or Update</string>
   </property>
   <widget class="QGroupBox" name="Small1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Select</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>84</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="VLayout">
      <item>
       <widget class="QRadioButton" name="InsertDep">
        <property name="text">
         <string>Insert</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="UpdateDep">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="DeleteDep">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="GoButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DepName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>          Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Mgr_SSN">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="DepartNum">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Return">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="LSSN">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Manager SSN</string>
    </property>
   </widget>
   <widget class="QLabel" name="LNumber">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DNumber</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Small2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Search By</string>
   </property>
   <widget class="QTextEdit" name="Search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="SName">
       <property name="text">
        <string>DName</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SNumber">
       <property name="text">
        <string>DNumber</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SMgrssn">
       <property name="text">
        <string>Manager SSN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="SearchTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show All</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
